analyses/create-subset-files/01-create_subset_files.Rmd

```diff
@@ -0,0 +1,370 @@
+---
+title: "Create subset files for CI"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: J. Taroni for ALSF CCDL
+date: 2019
+---
+
+In this notebook, we subset the files to include full features for 5 matched 
+participants.
+The output files are intended to be used for continuous integration.
+
+#### USAGE
+
+To run this from the command line, use:
+
+```
+Rscript -e "rmarkdown::render('analyses/create-subset-files/01-create_subset_files.Rmd', 
+                              clean = TRUE)"
+```
+
+_This assumes you are in the top directory of the repository._
+
+## Setup
+
+#### Packages and functions
+
+```{r}
+# R.utils is required to gzip files here
+if (!("R.utils" %in% installed.packages())) {
```

Comment:
> Do we wanna keep both installation with the Dockerfile as well as installation in this method just in case people are using these notebooks outside of a Docker image?

This was my plan. I don't currently see a downside to leaving it in, particularly when we will often be using a versioned `rocker` container during development.